app/partials/reimburse.html

```diff
@@ -0,0 +1,31 @@
+<h3>Reimbursement</h3>
+
+  <form ng-submit = "submit()">
+    <br>
+    <p>
```

Comment:
For general HTML style, it may be better to wrap everything in `<div>`s rather than `<p>`s